libs/ngx-mime/src/lib/viewer/recognized-text-content/recognized-text-content.component.spec.ts

```diff
@@ -25,6 +27,7 @@ describe('RecognizedTextContentComponent', () => {
         CanvasService,
         AltoService,
         MimeViewerIntl,
+        { provide: IiifContentSearchService, useClass: IiifContentSearchServiceStub },
```

Comment:
I don't think you need to specify the stub here. You can use the service directly.